src/hotspot/share/gc/shenandoah/shenandoahCardTable.hpp

```diff
@@ -38,7 +38,7 @@ class ShenandoahCardTable: public CardTable {
 
   virtual void initialize();
 
-  inline bool is_in_young(oop obj) const {
+  virtual inline bool is_in_young(oop obj) const {
```

Comment:
Why do we need virtual here? Is it overridden anywhere?

Reply:
Removing "virtual".